### PR TITLE
add xquad_r dataset

### PR DESCRIPTION
**Pull request by manandey**

### CODE REVIEW

File: datasets/xquad_r/xquad_r.py [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""TODO: Add a description here."""
+
+from __future__ import absolute_import, division, print_function
+import json
+import datasets
```
Comment (lhoestq) [inline datasets/xquad_r/xquad_r.py:19]:
```suggestion
from __future__ import absolute_import, division, print_function
import json

import datasets
```

File: datasets/xquad_r/xquad_r.py [hunk 2]
```diff
@@ -0,0 +1,141 @@
...
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""TODO: Add a description here."""
+
+from __future__ import absolute_import, division, print_function
+import json
```
Comment (lhoestq) [inline datasets/xquad_r/xquad_r.py:18]:
```suggestion
from __future__ import absolute_import, division, print_function

import json
```

File: datasets/xquad_r/README.md [hunk 3]
```diff
@@ -0,0 +1,153 @@
+# Dataset Card for [Dataset Name]
```
Comment (lhoestq) [inline datasets/xquad_r/README.md:1]: could you add the tags, i.e. the yaml part at the top of the dataset card ?

File: datasets/xquad_r/xquad_r.py [hunk 4]
```diff
@@ -0,0 +1,142 @@
...
+        super(XquadRConfig, self).__init__(version=datasets.Version("1.0.0", ""), **kwargs)
+        self.lang = lang
+
+
+# TODO: Name of the dataset usually match the script name with CamelCase instead of snake_case
+class XquadR(datasets.GeneratorBasedBuilder):
+    """TODO(xquad-r): Short description of my dataset."""
+
+    # TODO(xquad-r): Set up version.
+    VERSION = datasets.Version("1.1.0")
```
Comment (lhoestq) [inline datasets/xquad_r/xquad_r.py:70]:
```suggestion
    VERSION = datasets.Version("1.1.0")
    BUILDER_CONFIG_CLASS = XquadRConfig
```
we need that to specify that we're using a custom config

**Review by lhoestq — COMMENTED** [review]
really cool thanks !

Could you add the tags at the top of the dataset card ? more info here :
https://github.com/huggingface/datasets/blob/master/ADD_NEW_DATASET.md#tag-the-dataset-and-write-the-dataset-card

Also it looks like the dummy_data.zip files are quite heavy (almost 200KB per file). Can you try to reduce their sizes to a few KB ?

For exmaple for the dummy_data.zip file of the `ar` config, you can take a look inside the zip file and remove all the files related to the other configs (`de`, `el` etc.)

**Comment by lhoestq** [comment]
looks like this PR includes changes in many files than the ones for xquad_r, could you create a new branch and a new PR ?

**Comment by manandey** [comment]
Sure, I will close this then.

**Pull request closed without merging**